sklearn/metrics/tests/test_ranking.py

```diff
@@ -1650,6 +1650,30 @@ def test_top_k_accuracy_score_binary(y_score, k, true_score):
     assert score == score_acc == pytest.approx(true_score)
 
 
+@pytest.mark.parametrize('y_true, true_score, labels', [
+    (np.array([0, 1, 1, 2]), 0.75, [0, 1, 2, 3]),
```

Comment:
I think we should include this test case, but we would need to update the docstring. `y_true` in this case would be labeled as "multiclass"

Reply:
I am not sure what you mean. If I understand it correctly you already suggested that with the following change?
```
"""Test when labels and y_score are multiclass."""
```

Reply:
Yes that is what I meant.